core/drivers/crypto/caam/caam_ctrl.c

```diff
@@ -0,0 +1,156 @@
+// SPDX-License-Identifier: BSD-2-Clause
+/**
+ * @copyright 2017-2019 NXP
+ *
+ * @file    caam_ctrl.c
+ *
+ * @brief   CAAM Global Controller.\n
+ */
+
+/* Global includes */
+#include <initcall.h>
+#include <tee_api_types.h>
+
+#ifdef CFG_CRYPTO_DRIVER
+/* Driver Crypto includes */
+#include <drvcrypt.h>
+#endif
+
+/* Local includes */
+#include "common.h"
+#include "caam_jr.h"
+#include "caam_rng.h"
+#include "caam_pwr.h"
+#ifdef CFG_CRYPTO_HASH_HW
+#include "caam_hash.h"
+#endif
+
+/* Utils includes */
+#include "utils_mem.h"
+
+/* Hal includes */
+#include "hal_cfg.h"
+#include "hal_clk.h"
+#include "hal_ctrl.h"
+
+#define CTRL_DEBUG
```

Comment:
Prefer a CFG option

Reply:
rework debug trace in the driver to select the traces function of build CFG_CAAM_DBG 32 bits word